### DISCUSSION TITLE
“disqus-react” is not displayed when deployed on Vercel

### DISCUSSION THREAD

**Discussion by hassibmoddasser** [vercel/community#146] [Show & Tell]
I’m stuck with a bug in my Next js application, and it’s somewhat weird that I haven’t faced it before.

I’m using `disqus-react` in my application; it works fine in the local environment, but nothing is displayed as long as I deploy it on Vercel, and I see `Uncaught SyntaxError: Missing initializer in const declaration` error on the console.

I’d really appreciate your help! 💛

**Comment by Pieparker** [comment]
Hey there! Is this still an issue for you?

If so, please provide as much information as you can share – deployment IDs, commit SHAs, URLs and your account slug go a long way!

**Reply by hassibmoddasser** [reply]
Hey there! Yes, I still have the issue.

Everything is fine, and the `disqus-react` works in the local environment. Still, when deployed on Vercel, I get **three** `Uncaught SyntaxError: Missing initializer in const declaration` error on the console.

I'm also using the `react-share` package in the project and have the same problem.

Here's the website link so that you can check the errors: [https://merlino.agency/blog/clean-architecture-in-express-js-applications](https://merlino.agency/blog/clean-architecture-in-express-js-applications)

I'm using:

- `next`: 12.0.3
- `react`: 17.0.2
- `tailwindcss`: 3.0.1
- `typescript`: 4.4.4
- `disqus-react`: v1.1.2
- `react-share`: 4.4.0

**Reply by Pieparker** [reply]
Ah, it appears to be a client-side issue, given none of the resources `404`. This may be a question better suited to [the Disqus team](https://github.com/disqus/disqus-react) but we'd love to help if there's anything we can do!

**Reply by hassibmoddasser** [reply]
Thank you so much, @Pieparker!

I should consider opening the issue there. I really appreciate it.

**Reply by hassibmoddasser** [reply]
Hi there, this issue has been fixed. I think there was something wrong with Vercel.
